src/Features/CSharp/Portable/Completion/KeywordRecommenders/AbstractNativeIntegerKeywordRecommender.cs

```diff
@@ -19,6 +19,11 @@ internal abstract class AbstractNativeIntegerKeywordRecommender : IKeywordRecomm
 
         private static bool IsValidContext(CSharpSyntaxContext context)
         {
+            if (context.IsPreProcessorDirectiveContext)
+            {
+                return false;
+            }
```

Comment:
this is a workaround.  the problem is that one of the checks below is returning 'true' when it should not be.  the fix will need to go there.

Reply:
The root issue seems to be that `IsTypeContext` was true, because `IsGlobalStatementContext()` returned true.

The two conditional blocks I removed in the new commit have been there since the repo went public (says git blame), but that's where I would've put the check too (a preprocessor directive context is never a statement context, as far as my understanding of C# goes).